### PR TITLE
Vendor Fields relating to the Planet49 ruling

### PR DESCRIPTION
**Pull request by anderagakura**
Move the current vendor fields relating to the Planet 49 ruling from the existing Device Storage Access & Disclosure tech spec to this core spec

### CODE REVIEW

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 1]
```diff
@@ -1599,46 +1622,102 @@ Here is an annotated example of the GVL’s JSON format:
...
+
+The fields below provide Vendors with a method to disclose the length of time any vendor-specific information may be stored on a device and other summary information about device storage. Vendors also provide granular storage and access disclosures via a JSON file. CMPs display this information to consumers. This form of transparency took on urgency after a ruling by the Court of Justice of the European Union in Case C-673/17 _Planet49_.
+
+In its Planet49 judgment (available here), the Court of Justice of the European Union (CJEU) ruled that the requirements for informed consent under Article 5(3) of the ePrivacy Directive include the disclosure of “the duration of the operation of cookies and whether or not third parties may have access to those cookies” (para. 81).1 The CJEU clarified that the information provided must enable the user to determine the consequences of any consent and be sufficiently detailed so as to enable the user to understand the functioning of the cookies employed. This includes the requirement to provide information about the duration of the cookies and whether or not third parties may have access to those cookies.
```
Comment (tla-sirdata) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:1674]: Two missing links in this paragraph : 
- _here_ should point to http://curia.europa.eu/juris/document/document.jsf;jsessionid=DA9630A1B5B38A1B6EFD5C9CC1DCD815?text=&docid=218462&pageIndex=0&doclang=EN&mode=lst&dir=&occ=first&part=1&cid=8222720
- <sup>_[1]_</sup> should point to the added footnote 1

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md [hunk 2]
```diff
@@ -1599,46 +1622,102 @@ Here is an annotated example of the GVL’s JSON format:
...
+<table>
+ <tr><td>Field</td><td>Type</td><td>Description</td></tr>
+  <tr><td><code>usesNonCookieAccess</code></td><td>boolean</td><td>Indicates the vendor’s use of non-cookie storage and access to information already stored on a user’s device. True indicates non-cookie access is used. False indicates non-cookie storage and access to information already stored on a user's device is not used.</td></tr>
+ </table>
+ 
+ ##### deviceStorageDisclosureUrl
+ 
+ <table>
+ <tr><td>Field</td><td>Type</td><td>Description</td></tr>
+  <tr><td><code>deviceStorageDisclosureURL</code></td><td>boolean</td><td>This required field is a secure URL to a vendor-hosted resource that adheres to the structure, content and serving described by the Vendor Device Storage & Operational Disclosures <a href="https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/Additional-Vendor-Information-List-Specification.md">specification</a>.</td></tr>
```
Comment (tla-sirdata) [inline TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md:1718]: Based on [PR 313](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/pull/313), the URL for Vendor Device Storage & Operational Disclosures is https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/tree/master/TCFv2/Vendor%20Device%20Storage%20%26%20Operational%20Disclosures.md

**Review by tla-sirdata — CHANGES_REQUESTED** [review]
Some links must be added/updated

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md (hunk 1 above)
Comment (anderagakura) [reply TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md]: Update [here](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/pull/311/commits/4785f9d16f71484962024fcb407c036bb0eca30b)

File: TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md (hunk 2 above)
Comment (anderagakura) [reply TCFv2/IAB Tech Lab - Consent string and vendor list formats v2.md]: Update [here](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/pull/311/commits/4785f9d16f71484962024fcb407c036bb0eca30b)

**Review by HeinzBaumann — APPROVED** [review]

**Pull request merged**